test/solidity/Facets/AcrossFacetV3.t.sol

```diff
@@ -271,4 +276,23 @@ contract AcrossFacetV3Test is TestBaseFacet {
             true
         );
     }
+
+    function testRevert_WillFailIfBridgeDataReceiverDoesNotMatchWithAcrossData()
+        public
+    {
+        vm.startPrank(USER_SENDER);
+        usdc.approve(address(acrossFacetV3), bridgeData.minAmount);
+
+        validAcrossData.quoteTimestamp = uint32(block.timestamp - 100 days);
+
+        bridgeData.receiver = USER_REFUND; // does not match with USER_RECEIVER
```

Comment:
I prefer using a random address like address(0x123) rather than defined address that has a specific meaning in a different context.

But I am good if you want to leave it this way

Reply:
updated